ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/models/ExpenseItem.java

```diff
@@ -133,4 +134,12 @@ public void setReceiptPhoto(Bitmap photo) {
 	public Bitmap getReceiptPhoto(){
 		return this.receiptPhoto;
 	}
+
+	public boolean doesHavePhoto() {
+		return hasPhoto;
+	}
```

Comment:
[Beat you to it!](https://github.com/RostarSynergistics/ShinyExpenseTracker/pull/84#discussion_r26449875)